### PR TITLE
New Crowdin translations

### PR DESCRIPTION
**Pull request by Supamiu**
Enjoy!

### CODE REVIEW

File: apps/client/src/assets/i18n/es-ES.json [hunk 1]
```diff
@@ -1533,7 +1564,7 @@
     "Remove_from_basket": "Eliminar de la cesta",
     "Search": "Buscar recetas",
     "Generate_list": "Agregar a la lista",
-    "Description": "Introduzca los artículos que tiene y haga clic en \"Buscar recetas\" para obtener una lista de posibles recetas para fabricar y generar fácilmente una lista de fabricación de los resultados.",
+    "Description": "Input the items you have and click \"Find recipes\" to get a list of possible recipes to craft and easily generate a crafting list from the results.",
```
Comment (noshlag) [inline apps/client/src/assets/i18n/es-ES.json]: This was in spanish.  Why did you change it to english?

File: apps/client/src/assets/i18n/fr-FR.json [hunk 2]
```diff
@@ -1533,7 +1564,7 @@
     "Remove_from_basket": "Retirer du panier",
     "Search": "Trouver des recettes",
     "Generate_list": "Ajouter à une liste",
-    "Description": "Saisissez les objets que vous avez et cliquez sur « Trouver des recettes » pour obtenir une liste de recettes possibles pour fabriquer et générer une liste d'artisanat à partir des résultats.",
+    "Description": "Input the items you have and click \"Find recipes\" to get a list of possible recipes to craft and easily generate a crafting list from the results.",
```
Comment (noshlag) [inline apps/client/src/assets/i18n/fr-FR.json]: This was in french, why did you change it to english?

File: apps/client/src/assets/i18n/ja-JP.json [hunk 3]
```diff
@@ -1533,7 +1564,7 @@
     "Remove_from_basket": "制作リストから削除",
     "Search": "レシピ検索",
     "Generate_list": "リストに追加",
-    "Description": "現在お持ちの素材の数量を入力し \"レシピ検索\" をクリックすると現在の素材で制作可能な品目がご確認することができ、望みな品目をリストに作成するのができます。",
+    "Description": "Input the items you have and click \"Find recipes\" to get a list of possible recipes to craft and easily generate a crafting list from the results.",
```
Comment (noshlag) [inline apps/client/src/assets/i18n/ja-JP.json]: Same comment as above

File: apps/client/src/assets/i18n/ko-KR.json [hunk 4]
```diff
@@ -1533,7 +1564,7 @@
     "Remove_from_basket": "제작 목록에서 제거",
     "Search": "제작법 찾기",
     "Generate_list": "목록에 추가",
-    "Description": "현재 가지고 있는 재료의 수량을 입력하고 \"제작법 찾기\" 를 클릭하면 현재 재료로 제작이 가능한 품목들을 확인할 수 있으며, 원하는 품목들을 목록으로 만들 수 있습니다.",
+    "Description": "Input the items you have and click \"Find recipes\" to get a list of possible recipes to craft and easily generate a crafting list from the results.",
```
Comment (noshlag) [inline apps/client/src/assets/i18n/ko-KR.json]: Same comment as above

File: apps/client/src/assets/i18n/ru-RU.json [hunk 5]
```diff
@@ -1533,7 +1564,7 @@
     "Remove_from_basket": "Убрать из корзины",
     "Search": "Найти рецепты",
     "Generate_list": "Добавить в список",
-    "Description": "Введите предметы, которые у вас есть, и нажмите \"Найти рецепты\", чтобы получить список возможных рецептов и создать список для крафта из результатов.",
+    "Description": "Input the items you have and click \"Find recipes\" to get a list of possible recipes to craft and easily generate a crafting list from the results.",
```
Comment (noshlag) [inline apps/client/src/assets/i18n/ru-RU.json]: same comment as above

File: apps/client/src/assets/i18n/zh-CN.json [hunk 6]
```diff
@@ -1533,7 +1564,7 @@
     "Remove_from_basket": "从愿望篮中删除",
     "Search": "查找配方",
     "Generate_list": "添加到清单",
-    "Description": "输入您拥有的物品并点击“查询配方”，可以得到一系列的潜在可制作配方，并能够从其结果中生成相应的清单。",
+    "Description": "Input the items you have and click \"Find recipes\" to get a list of possible recipes to craft and easily generate a crafting list from the results.",
```
Comment (noshlag) [inline apps/client/src/assets/i18n/zh-CN.json]: same comment as above

File: apps/client/src/assets/i18n/fr-FR.json (hunk 2 above)
Comment (Supamiu) [reply apps/client/src/assets/i18n/fr-FR.json]: This has been done by crowdin, because I fixed a typo in the original english translation and it reset the other languages to that as it can't guess if the change was minor or major for the meaning of the sentence.
